### PR TITLE
Added sensitive property

### PR DESCRIPTION
**Pull request by pasimko**

### CODE REVIEW

File: resources/osl_nextcloud.rb [hunk 1]
```diff
@@ -91,7 +92,7 @@
       --database-pass #{new_resource.database_password} \
       --admin-user #{new_resource.nextcloud_admin_user} \
       --admin-pass #{new_resource.nextcloud_admin_password}"
-    sensitive true
+    sensitive sensitive
```
Comment (ramereth) [inline resources/osl_nextcloud.rb:95]:
```suggestion
    sensitive new_resource.sensitive
```

**Review by ramereth — CHANGES_REQUESTED** [review]

**Review by ramereth — APPROVED** [review]
minor bump

**Comment by pasimko** [comment]
!bump minor ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `main` and has automatically performed a minor-level version bump to v1.1.0.  Have a nice day!

**Pull request merged**